source/tutorials/feature-classifier.rst

```diff
@@ -0,0 +1,71 @@
+feature-classifier tutorial
+===========================
+
+.. note:: This guide assumes you have performed the steps in the :doc:`install guide <../install>`.
+
+This tutorial will demonstrate how to retrain the ``feature-classifier`` for a particular dataset. We will retrain the `Naive Bayes`_ classifier using `Greengenes`_ reference sequences and classify the representative sequences from the `Moving Pictures`_ dataset.
+
+We will download and create several files, so first create a working directory.
+
+.. command-block::
+    :no-exec:
+
+    mkdir feature-classifier-tutorial
+    cd feature-classifier-tutorial
+
+Obtaining and importing reference data sets
+-------------------------------------------
+
+Two elements are required for training the classifier: the reference sequences and the corresponding taxonomic classifications. To reduce computation time for this tutorial we will use the relatively small `Greengenes`_ 13_8 85% OTU data set. The next step will require aligned reference sequences. The ones that we will download here are already aligned.
+
+.. note:: The aligned sequences that are available in the standard Greengenes data sets have had some columns removed, which would adversely affect the performance of the classifier. If you are starting from scratch, download the unaligned reference sequences and align them yourself.
+
+We will also download the representative sequences from the `Moving Pictures`_ tutorial to check that the classifier classifies.
+
+.. command-block::
+
+    curl -sL https://zenodo.org/record/179385/files/aligned_85_otu_sequences.fasta.gz > aligned_85_otu_sequences.fasta.gz
```

Comment:
@gregcaporaso should we put these files on data.qiime2.org so they are versioned?